ext/json/php_json_encoder.h

```diff
@@ -35,6 +35,6 @@ static inline void php_json_encode_init(php_json_encoder *encoder)
 
 zend_result php_json_encode_zval(smart_str *buf, zval *val, int options, php_json_encoder *encoder);
 
-int php_json_escape_string(smart_str *buf, const char *s, size_t len, int options, php_json_encoder *encoder);
+zend_result php_json_escape_string(smart_str *buf, const char *s, size_t len, int options, php_json_encoder *encoder);
```

Comment:
I'm not sure if it's a good idea to change prototype of exported function in the stable branch.
It may be used by some third party extensions.

The rest looks fine.

Reply:
I merge it without the JSON commit